test/functional/fixtures/es_archiver/deprecations_service/mappings.json

```diff
@@ -423,7 +423,10 @@
         "number_of_shards": "1",
         "priority": "10",
         "refresh_interval": "1s",
-        "routing_partition_size": "1"
+        "routing_partition_size": "1",
+        "mapping": {
+          "total_fields": { "limit": 1500 }
```

Comment:
Here and in all the other mappings.json files I explicitly set the field limit to 1500. When FTR has an esArchive with the index name `.kibana_$KIBANA_PACKAGE_VERSION_001` this creates an index with the current version number like `.kibana_8.5.0_001`. Because the index is already using the latest version number no migrations are applied and so the migration service will not update the field limit of this index.

So any time esArchiver creates such an "already migrated to the latest version" index we need to explicitly set the field limit. Because most tests don't delete the .kibana index once it exists, but merely "cleans" it by deleting all current saved objects, these archives can break other test suites that didn't even load the archive.

The alternative is to always delete the .kibana index for every test suite, but this is simply too time consuming and adds an incredible amount of time to each CI run.

Reply:
thx @rudolf for taking care of this!

Reply:
Thanks @rudolf!